src/app/current/current.component.html

```diff
@@ -1,5 +1,53 @@
-<p>current works!</p>
+<!-- 
 <span>{{requestServise.currentWeather | json}}</span>
-<p>{{requestServise.currentWeather?.name}}</p>
-<p>{{requestServise.currentWeather?.main.temp}}</p>
-<p>{{requestServise.currentWeather?.weather[0].main}}</p>
+<span>{{requestServise.currentWeatherData | json}}</span>
+-->
+<table class="table table-bordered my-5">
+  <thead>
+    <tr>
+      <th scope="col">Location</th>
+      <th scope="col">Weather icon</th>
+      <th scope="col">Weather</th>
+      <th scope="col">Temperature [celsius]</th>
+      <th scope="col">Wind speed [meter/sec]</th>
+      <th scope="col">Cloudness [%]</th>
+      <th scope="col">Pressure [hPa]</th>
+      <th scope="col">Humidity [%]</th>
+      <th scope="col">Sunrise [hh:mm]</th>
+      <th scope="col">Sunset [hh:mm]</th>
+      <th scope="col">Rain [mm/1h]</th>
+      <th scope="col">Snow [mm/1h]</th>
+      <th scope="col">Favourite location</th>
+      <th scope="col">Cancel</th>
```

Comment:
I think table header for this column is unnecessary, moreover I suggest that all action buttons should have be in a single column for example as a button group.